### PR TITLE
Chore: Bootstrap React Native. Added dependencies, Redux and Router

### PR DESCRIPTION
**Pull request by johnandblue**

### CODE REVIEW

File: App/reducers/reducers.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      return {};
+    default:
+  }
+  return state;
+}
+
+const songs = (state = [], action) => {
+  switch (action.type) {
+    case 'GET_SONGS_SUCCESS':
+      return [...action.response];
```
Comment (arol) [inline App/reducers/reducers.js:47]: Why not only `action.response`?

File: App/reducers/reducers.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  }
+
+  return state;
+}
+
+const user = (state = {}, action) => {
+  switch (action.type) {
+    case 'LOGIN_SUCCESS':
+      return {...action.response};
```
Comment (arol) [inline App/reducers/reducers.js:36]: Why not only `action.response`

**Review by arol — APPROVED** [review]
👏 👍

File: App/reducers/reducers.js (hunk 2 above)
Comment (johnandblue) [reply App/reducers/reducers.js:36]: Fixed. Now only returning response

File: App/reducers/reducers.js (hunk 1 above)
Comment (johnandblue) [reply App/reducers/reducers.js:47]: Fixed. Now only returning response

**Pull request merged**